src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Modal from '../Modal/Modal';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+
+export const ModalCancelEvent = ({ cancelMessage }) => {
+  return (
+    <div>
```

Comment:
Do you need this outer `div`? If not, remove it.

Reply:
removed